### PR TITLE
Release 2.28.1

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,30 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [2.29.0](https://github.com/serverless/serverless/compare/v2.28.0...v2.29.0) (2021-03-02)
+
+### Features
+
+- **Variables:** Resolve all env variables with new resolver ([#9040](https://github.com/serverless/serverless/pull/9040)) ([c1d8b58](https://github.com/serverless/serverless/commit/c1d8b58ed8a5a7a91d9dfa28536a9c0d997b809b)) ([Mariusz Nowak](https://github.com/medikoo)
```
Comment (medikoo) [inline CHANGELOG.md:9]: I would put it to _maintainance improvements_ as it doesn't bring any real change to the user (it was also marked as _refactor_).

Also then I'd release it as v2.28.2

**Review by medikoo — CHANGES_REQUESTED** [review]
Thanks @pgrzesik ! Still I think it's more a patch release, see my comments

File: CHANGELOG.md (hunk 1 above)
Comment (pgrzesik) [reply CHANGELOG.md]: Thanks :bow: Change to patch release v2.28.1 as it's the next version number

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**